packaging/nnstreamer.spec

```diff
@@ -178,8 +178,9 @@ enable_tf=true
 %else
 enable_tf=false
 %endif
+enable_torch=false
 
-meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir=%{_libdir} --bindir=%{nnstexampledir} --includedir=%{_includedir} -Dinstall-example=true -Denable-tensorflow=${enable_tf} %{api} -Denable-env-var=false build
+meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir=%{_libdir} --bindir=%{nnstexampledir} --includedir=%{_includedir} -Dinstall-example=true -Denable-tensorflow=${enable_tf} -Denable-pytorch=${enable_torch} %{api} -Denable-env-var=false build
```

Comment:
Use %, not $ in .spec script.

Actually, ${enable_tf} is incorrect as well.

Reply:
Done
Will update enable_tf in a different PR